examples/notebooks/05_market_comparison.ipynb

```diff
@@ -0,0 +1,363 @@
+      "text/plain": [
+       "<Figure size 800x550 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# second plot\n",
+    "sql = \"\"\"\n",
+    "SELECT\n",
+    "time_bucket('1h',datetime) AS \"time\",\n",
+    "sum(power) AS \"market_dispatch\",\n",
+    "market_id,\n",
+    "um.technology\n",
+    "FROM market_dispatch md\n",
+    "join power_plant_meta um on um.index = md.unit_id and um.simulation = md.simulation\n",
+    "WHERE\n",
+    "md.simulation = 'example_02c_ltm_case'\n",
+    "GROUP BY 1, market_id, technology\n",
+    "ORDER BY technology, market_id desc, 1\n",
+    "\"\"\"\n",
+    "\n",
+    "df = pd.read_sql(sql, engine, index_col=\"time\")\n",
+    "# fig, ax = plt.subplots(figsize=(8,6))\n",
+    "series = []\n",
+    "for label, sub_df in df.groupby([\"market_id\", \"technology\"]):\n",
+    "    lab = \"-\".join(label)\n",
+    "    lab = lab.replace(\"LTM_OTC\", \"LTM\")\n",
+    "\n",
+    "    if \"lignite\" not in lab and \"nuclear\" not in lab:\n",
+    "        continue\n",
+    "    group_sum = sub_df.market_dispatch.groupby(\"time\").sum()\n",
+    "    group_sum.name = lab\n",
+    "    series.append(group_sum.resample(\"1h\").ffill())\n",
+    "\n",
+    "ddf = pd.DataFrame(series)\n",
+    "ddf = ddf.T.fillna(method=\"ffill\")\n",
+    "\n",
+    "ddf = ddf[sorted(ddf.columns, reverse=True)]\n",
+    "ddf = ddf.fillna(0)\n",
+    "ddf /= 1e3\n",
+    "base = ddf[ddf.columns[0]] * 0\n",
+    "for col in ddf.columns:\n",
+    "    line = base + ddf[col]\n",
+    "    c = (0.3, 0.2, 0.6, 0.8) if \"nuclear\" in col else \"g\"\n",
+    "    alpha = 0.8 if \"LTM\" in col else 0.6\n",
+    "    plt.fill_between(line.index, line, base, alpha=alpha, label=col, color=c)\n",
+    "    base += ddf[col]\n",
+    "plt.ylabel(\"Hourly dispatch power [$GW$]\")\n",
+    "plt.xlabel(\"Datetime\")\n",
+    "plt.xticks(rotation=25)\n",
+    "plt.legend()\n",
+    "savefig(\"overview-dispatch.png\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here one can see that the dispatch stays about the same, but a large amount has been traded as a base band on the LTM while the daily variances are traded on the EOM.\n",
+    "\n",
+    "This also brings us to the end of this short tutorial on multiple markets."
```

Comment:
here, a short summary of what people learned (in this case how to adjust configs for an extra market), and how to visualize the results. I believe what is also missing here is the part about bidding strategies, how exactly are they condifured for 2 markets. Maybe it makes sense to have a section in the tutorial on that? Just a brief mention of what is the bidding strategy, how it is called, and how exactly is it given via the inputs

Reply:
I would agree on that. This is a great opportunity to showcase the interplay of different bidding strategies.

Reply:
@maurerle yes, exactly, this is also pretty cool, and a good contribution of the paper and a nice insight people can get from the tutorial